src/server/pfs/server/obj_block_api_server.go

```diff
@@ -574,11 +698,11 @@ func (s *objBlockAPIServer) objectGetter(ctx groupcache.Context, key string, des
 
 func (s *objBlockAPIServer) tagGetter(ctx groupcache.Context, key string, dest groupcache.Sink) error {
 	splitKey := strings.Split(key, ".")
-	if len(splitKey) != 2 {
+	if len(splitKey) != 3 {
 		return fmt.Errorf("invalid key %s (this is likely a bug)", key)
 	}
 	prefix := splitKey[0]
-	tag := &pfsclient.Tag{Name: strings.Join(splitKey, "")}
+	tag := &pfsclient.Tag{Name: strings.Join(splitKey[:len(splitKey)-1], "")}
```

Comment:
It looks like we're calling strings.Join on a single string here. Is that correct?